pallets/registrar/src/lib.rs

```diff
@@ -821,6 +857,14 @@ pub mod pallet {
                 Self::deposit_event(Event::ParaIdDeregistered { para_id });
                 // Cleanup immediately
                 Self::cleanup_deregistered_para_id(para_id);
+                if let Err(id) = BufferedParasToDeregister::<T>::try_mutate(|v| v.try_push(para_id))
+                {
+                    log::error!(
+                        target: LOG_TARGET,
```

Comment:
@tmpolaczyk what do you think about this syntax? Would you suggest to use something else instead?

Reply:
I changed it in 2f0a142be0c68c02c15fef5aa01c3f7de99f174a and now I'm returning a custom DispatchError (I needed to convert the internal error).